contributing/ci-omero.txt

```diff
@@ -36,17 +36,19 @@ OMERO jobs
         * :term:`OMERO-DEV-merge-deploy`
         * :term:`OMERO-DEV-breaking-deploy`
 
+    -   * Deploys the merge OMERO web
+        * :term:`WEB-DEV-merge-deploy`
+        * :term:`WEB-DEV-breaking-deploy`
+
```

Comment:
Do we also need to list WEB-DEV-latest-deploy and WEB-DEV-integration-deploy in this table?